app/controllers/saml_idp_controller.rb

```diff
@@ -21,6 +21,7 @@ def auth
 
     return redirect_to idv_url if needs_idv
 
+    delete_branded_experience
```

Comment:
Since we are only using this one one place, I'd define this method in this controller rather than making it available to all controllers upront